docker-compose.yml

```diff
@@ -27,7 +27,9 @@ services:
       - 'pktfwdr:/var/pktfwd'
 
   helium-miner:
-    image: "nebraltd/hm-miner:6aacef4c2fcab92aee5fbdcc5ac6ca5927b97421"
+    image: "nebraltd/hm-miner:b404b50721f87c0f70e60bb8f5ec0a75d05b2445"
+    expose:
+      - "4467"
```

Comment:
What's this for?

Reply:
Good catch. That's the RPC interface. That should probably only be exposed locally.

Reply:
This is only exposed within the Docker internal network and allows the hm-diag / hm-config containers to reach the JSON RPC API. Expose is used to expose ports internally. Ports is used to expose ports to the host / external network.